README.md

```diff
@@ -35,6 +35,16 @@ pip install awsiotsdk
 pip install ./aws-iot-device-sdk-python-v2
 ```
 
+## Install for Raspberry Pi ([buster](https://www.raspberrypi.org/downloads/raspbian/))
+
+```
+sudo apt-get update
+sudo apt-get install cmake
+sudo apt-get install libssl-dev
```

Comment:
Have you tried building without `libssl-dev` installed?
I'm not sure myself whether it is or isn't, but don't want any unnecessary steps here.
I don't have access to a pi right now (quarantine) so can't test myself

Reply:
Sorry, usually I'd try this all out myself, but I brought my pi to the office before quarantine and don't have access to it right now

Reply:
`LibCrypto` is required to build s2n. Thats why `libssl-dev` need to install.